scaleway/helpers_lb.go

```diff
@@ -17,6 +17,7 @@ import (
 const (
 	LbWaitForTimeout   = 10 * time.Minute
 	defaultLbLbTimeout = 10 * time.Minute
+	retryLbIPInterval  = 1 * time.Minute
```

Comment:
it seems very long to wait a minute per attempt. Could we make it smaller like 5 seconds or something like that?

Reply:
Yes let me try this and I keep you up

Reply:
The test were successful